pkg/utils/has/components.go

```diff
@@ -80,6 +80,24 @@ func (h *HasController) GetComponentPipelineRun(componentName string, applicatio
 	return nil, fmt.Errorf("no pipelinerun found for component %s", componentName)
 }
 
+// GetAllPipelineRunInNameSpace returns the pipeline for a given application in a namespace
+func (h *HasController) GetAllPipelineRunInNameSpace(applicationName, namespace string) (*v1beta1.PipelineRunList, error) {
```

Comment:
thinking about more suitable name for this method.. wdyt about `GetAllPipelineRunsForApplication`?

Reply:
Agree. Changed it now.